pkg/webhook/policy.go

```diff
@@ -182,11 +181,21 @@ func (h *validationHandler) Handle(ctx context.Context, req admission.Request) a
 	}
 
 	res := resp.Results()
-	msgs := h.getDenyMessages(res, req)
-	if len(msgs) > 0 {
-		vResp := admission.ValidationResponse(false, strings.Join(msgs, "\n"))
-		if vResp.Result == nil {
-			vResp.Result = &metav1.Status{}
+	denyMsgs, warnMsgs := h.getAdmissionMessages(res, req)
```

Comment:
nit: should we call this `getValidationMessages` now? Mutation is technically also an admission request.

Reply:
updated